src/viur/core/utils/__init__.py

```diff
@@ -1,38 +1,20 @@
 import hashlib
 import hmac
 import warnings
-
 import logging
-import secrets
-import string
-import typing
 from base64 import urlsafe_b64encode
 from datetime import datetime, timedelta, timezone
 from typing import Any, Union, Optional
 from urllib.parse import quote
-
 from viur.core import current, db
 from viur.core.config import conf
-
+from . import string
```

Comment:
I would prefer to have a
```py
from .string import *
```
with `__all__` configured so I can just import everything from `viur.core.utils`. This would also be no problem if we change the structure again in the future.

Reply:
Hmmm... okay I've now introduced `parse.bool`, which would introduce `bool` as a function - no good idea ;-)

Reply:
Well, okay then…